ci/requirements-3.5_DOC_BUILD.sh

```diff
@@ -6,6 +6,6 @@ echo "[install DOC_BUILD deps]"
 
 pip install pandas-gbq
 
-conda install -n pandas -c conda-forge feather-format
+conda install -n pandas -c conda-forge feather-format nbsphinx pypandoc
```

Comment:
What do you need `pypandoc` for?
Wouldn't `pandoc` be more appropriate instead?
https://anaconda.org/conda-forge/pandoc